code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -432,7 +432,22 @@
 
 //Overriding this will stop a number of headaches down the track.
 /mob/living/silicon/pai/attackby(obj/item/weapon/W as obj, mob/user as mob, params)
-	if(W.force)
+	if(istype(W, /obj/item/stack/nanopaste))
+		var/obj/item/stack/nanopaste/N = W
+		if (stat == DEAD)
+			user << "<span class='danger'>The [src] is beyond help, at this point.</span>"

```

Comment:
It should be `\The [src]`. Otherwise it will look like `The N.E.K.O. is beyond help, at this point.`